tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -7927,13 +7939,16 @@ <h2>Show similarity to selected datapoint</h2>
           return {step: origValue, scalar: origInferenceScore};
         },
 
-        deletePdPlotSpinner: function(featureName) {
+        deletePdPlotProgress: function(featureName) {
           const container = this.featureContainerByName(featureName);
-          deleteElement(container.querySelector('paper-spinner-lite'));
+          const elements = container.querySelectorAll('paper-progress');
```

Comment:
why was this change necessary?

Reply:
if you open and close PD plot container fast you can get multiple loading bars created. this just helps ensure if this happens, that when the PD plot is created, all the loading bars go away, not just one of them (same thing was happening with loading circles before this change).

Reply:
good catch!